### ISSUE TITLE
Container not actually working

### ISSUE THREAD

**Issue by Welved** [ABeltramo/cloudtube-docker#8]
I have followed the write up that is available on docker hub and installed this container like any other container using portainer.

It does appear to have installed and it is running, but I am unable to actually visit either 10412 or 3000. Here is a screenshot of how I have installed it https://imgur.com/a/ZyySc0o

**Comment by ABeltramo** [comment]
You need to run two containers for Cloudtube to work: one is the frontend serving the HTTP clients and the other one is the server dealing with getting data from Youtube.  
Are you running both? Can you provide some more info and logs?

**Comment by Welved** [comment]
No, I figured the container writes to the same directory; so I decided to set their mount points and add the ports in one container.

Apparently that doesn't work.

**Comment by TheKatariya** [comment]
I ran the docker compose found here: https://github.com/ABeltramo/cloudtube-docker/tree/main/examples The containers start, but no logs are produced, and the cloudtube is not accessible at 3001. I would hare logs, but not logs are produced...
